dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd

```diff
@@ -11,6 +11,9 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
+javac.source: 1.8
```

Comment:
Should this have a Require-Capability 1.8 as well?

Reply:
also, I don't think it's necessary to put javac.source/target in here because there are no java files in this project

Reply:
It already has the Require-Capability 1.8 in it.